### DISCUSSION TITLE
Découverte dongles déportés SMLIGHT - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#44] [Comments]
# Découverte dongles déportés SMLIGHT - Haade.fr

SMLight nous fourni une gamme complète de coordinateurs/dongle Zigbee déporté et très complet, une installation simplifiée, une interface web et le tout compatible Zigbee2mqtt et ZHA.

https://haade.fr/fr/blog/decouverte-gamme-dongle-zigbee-matter-thread-deporte-smlight-slzb-06-et-slzb-07

**Comment by prodoff** [comment]
Merci pour ce test 👍

**Reply by haade-administrator** [reply]
De rien, il me reste juste à tester le vpn

**Comment by Mushupm** [comment]
Bonjour,
J'utilise un slzb-06 et j'ai un problème de portée pour les équipements du local piscine dans le jardin.
Avez vous fait un comparatif de portée par rapport au sonoff E?

Il y a maintenant le sonoff MAX pour lequel vous indiquez une portée supérieure au modèle nabu casa.
Comment se compare t'il par rapport au slzb-06?

**Reply by haade-administrator** [reply]
bonjour rien a été comparé au slzb-06 mais celui-ci a une portée équivalente aux anciens sonoff-E donc inférieur au dongle max

**Reply by Mushupm** [reply]
Bonjour,
Merci pour cette réponse qui me permet d'envisager le dongle Max.
